docs/data-tunnel/route-types/04-router.md

```diff
@@ -0,0 +1,118 @@
+# Router Route
```

Comment:
can we add some link to their docs?  like which chain they support or something.

Reply:
we have here on line 17 https://docs.routerprotocol.com/networks/supported-chains/